apps/api/v2/src/ee/bookings/2024-08-13/outputs/get-cal-video-sessions.ts

```diff
@@ -0,0 +1,99 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { Expose } from "class-transformer";
+import { IsEnum, ValidateNested, IsNumber, IsString, IsBoolean, IsArray, IsOptional } from "class-validator";
+
+import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+
+class Participant {
+  @ApiProperty({ type: String, nullable: true })
+  @IsOptional()
+  @Expose()
+  user_id!: string | null;
```

Comment:
For outputs we use `camelCase` so need to have `userId` and same for rest of the keys. I reccommend creating a new service `bookings-video-sessions-output.service.ts` that takes video session of cal and returns the type defined here and then `getCalVideoSessions` uses the output service.